### DISCUSSION TITLE
info: Custom config file missing or YAML format invalid.

### DISCUSSION THREAD

**Discussion by KurtKobalt** [danny-avila/LibreChat#2453] [Troubleshooting]
I have a clean installation through Docker, and when I rename librechat.example.yaml to librechat.yaml, the configuration file is not loaded and I can see "info: Custom config file missing or YAML format invalid." in the dockers log. I've also tried to change the CONFIG_PATH in the .env to point to another config file, but it didn't work either. 

Anyone knows what can be happening?
Thanks!

**Comment by danny-avila** [comment]
The example may need updating as is “invalid”. I recommend making a new file based on your needs

**Reply by KurtKobalt** [reply]
I tried with minimal code (literally only "version: 1.0.5") and the same error pops up. I'm just restarting the container, should I be doing something else? Can it be cached somewhere?

**Reply by danny-avila** [reply]
@KurtKobalt I just tested the example file and it works fine. I think you need to use a docker override file since you didn't mention it.

https://docs.librechat.ai/install/configuration/custom_config.html#docker-setup

**Reply by KurtKobalt** [reply]
thank you @danny-avila, I did override the docker compose. Not sure what was wrong but I started from scratch and now it's working fine. Thanks!
